### ISSUE TITLE
Decompressing nsz won't work if the extension is uppercase.

### ISSUE THREAD

**Issue by seedds** [julesontheroad/NSC_BUILDER#215]
No error showed, and no decompressed file either.

**Comment by julesontheroad** [comment]
Yeah there's a filter that prevents non supported formats to  enter on the list, the reason is the list function is made to scan folders and subfolders too so it will ignore non supported formats on those. Then several functions aren't made to assume the extension may be on uppercase.
For error i could add a `X number of files were added to list i guess`. For supporting those extensions i have to check the extension of what needs to be modified.
